app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java

```diff
@@ -78,7 +78,6 @@ protected void onCreate(Bundle savedInstanceState) {
 
         binding.navView.getMenu().setGroupVisible(R.id.subMenu, false);
         fragmentManager = getSupportFragmentManager();
-
```

Comment:
Unneeded change

Reply:
The rotation is not disabled. Please check again
Here is the screenshot of the activity when the device is rotated.

![screenshot_20180225-101436](https://user-images.githubusercontent.com/23053724/36638230-2ed34824-1a15-11e8-960c-69da588a2fc9.png)

Reply:
I will check the other changes made.